lib/data/use_case/anime/get_top_anime.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+import '../../../presentation/feature/news/models/anime_news_item_model.dart';
+import '../../model/api/request/get_top_anime_request.dart';
+import '../../providers/my_anime_list_repository_provider.dart';
+
+part 'get_top_anime.g.dart';
+
+@riverpod
+Future<List<AnimeNewsItemModel>> topAnime(
```

Comment:
```suggestion
Future<List<AnimeNewsItemModel>> getTopAnimeUseCase(
```
I think we should make the file with the same name?

Reply:
This is only an opinion, for the template I think the anime folder is not necessary, if not the folder tree is too big and is difficult to understand

Reply:
As for the renaming suggestion from `topAnime` to `getTopAnimeUseCase`, I understand what you're saying. However this function is special in that it is annotated with `@riverpod` and thus Riverpod Generator will make it available globally as `topAnimeProvider` (see `news_page.dart`), which makes sense. But if it is renamed to `getTopAnimeUseCase` the provider will be `getTopAnimeUseCaseProvider` which I think is not accurate.

Reply:
As for the `anime` folder's necessity. Even though the template is not complex, it is the skeleton of a larger project, which will perhaps require a folder for each top level API path (for example, /user /products /news). I see what you're saying but do you have a suggestion then to make the folder tree easier to understand?